Exercises/numbers/numbers.js

```diff
@@ -0,0 +1,53 @@
+//1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+//const decimal = new Number('99');
+
+const binary = new Number('1100011');
```

Comment:
I think it will not be binary, will be just 1100011 in decimal

Reply:
got it! updated to 
const binary = '0000000001111111';
const octal = '0x7F';
const hexadecimal = '7F';